{{cookiecutter.project_slug}}/clients/web/vue3/src/views/Signup.vue

```diff
@@ -92,8 +93,7 @@ export default {
       unwrappedForm.validate()
       if (!unwrappedForm.isValid) return
 
-      User.api
-        .registerUser({
+      userApi.create({
```

Comment:
`csc` missing here?

Reply:
These are just built-ins so they live at the root of the api object

https://github.com/thinknimble/tn-models-fp#built-in-methods